fastlane/Matchfile

```diff
@@ -6,7 +6,6 @@ platform "ios"
 type "appstore"
 app_identifier ["com.duckduckgo.mobile.ios", "com.duckduckgo.mobile.ios.ShareExtension", "com.duckduckgo.mobile.ios.OpenAction2", "com.duckduckgo.mobile.ios.Widgets", "com.duckduckgo.mobile.ios.NetworkExtension"]
 generate_apple_certs false
-template_name "Default Web Browser iOS (Dist)"
```

Comment:
This forces it to use the wrong profile.  The entitlements are now driven by the app id configuration.